### PR TITLE
Add the changelog file

### LINKED ISSUE #15: Add a changelog
Follow the pattern here: https://keepachangelog.com/en/1.0.0/

### PR DESCRIPTION
**Pull request by arturparkhisenko**
Changelog:
- Added the changelog file, closes #15 

And as a part of the Hacktoberfest month, I would appreciate and like to ask you to add all the necessary labels(tags), please.
It is `hacktoberfest` topic to the repository and `hacktoberfest-accepted` if PR was accepted.

### CODE REVIEW

**Comment by SeanMcP** [comment]
@arturparkhisenko This looks good! Did you use a tool to generate this file or is it based on commit messages?

This does a better job of tracking changes than I could have done!

**Comment by arturparkhisenko** [comment]
@SeanMcP I did it manually, what I found is working best for me, if the repo has tags, I would use a URL like this https://github.com/SeanMcP/a11y-react-emoji/compare/v1.1...v1.1.2
You can get this URL by clicking the `Compare` button when you're looking at the [releases](https://github.com/SeanMcP/a11y-react-emoji/releases) page.

**Comment by arturparkhisenko** [comment]
@SeanMcP I believe there are some tools to automate it if we'll be able to follow some format, there's a [lot](https://github.com/marketplace?type=actions&query=changelog) of them.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
```
Comment (SeanMcP) [inline CHANGELOG.md:7]: Add an `## [Unreleased` section here with `- Added `CHANGELOG.md`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- Security improvements with the updated Yarn lock-file
+
+## [1.0.1] - 2019-05-18
+
+- Changed React peer dependency version to allow any minor or patch ranges
+
+## [1.0.0] - 2019-01-19
+
+The first official release of `a11y-react-emoji` 🎉
```
Comment (SeanMcP) [inline CHANGELOG.md]: For consistency, prepend a `- ` to this line.

**Review by SeanMcP — CHANGES_REQUESTED** [review]
Like I mentioned above, this is great work. Two small requests for change:

**Review by SeanMcP — APPROVED** [review]
Looks good! Thanks for making those changes 👍

**Comment by arturparkhisenko** [comment]
@SeanMcP Can you also add the `hacktoberfest-accepted` tag? Thanks

**Pull request merged**